cmd/meroxa/root/apps/deploy.go

```diff
@@ -482,6 +482,12 @@ func (d *Deploy) getAppImage(ctx context.Context) (string, error) {
 		return "", nil
 	}
 
+	// Prevent creation of function in an Env
+	if d.flags.Environment != "" {
+		d.logger.StopSpinnerWithStatus("\t", log.Failed)
+		return "", errors.New("cannot deploy an application with a functions to an environment")
```

Comment:
sorry i didn't review sooner.  just noticed that functions is plural here with a singular article. could it be "one or more functions" so it doesn't have to be rewritten when we support more?

Reply:
Nice catch! I think for now it's best to only mention one function, so I've went ahead and switched `functions` to singular `function` https://github.com/meroxa/cli/pull/627